pkg/services/queryhistory/models.go

```diff
@@ -53,6 +62,10 @@ type QueryHistoryResponse struct {
 	Result QueryHistoryDTO `json:"result"`
 }
 
+type QueryHistorySearchResponse struct {
```

Comment:
Are you planning to include metadata about the search like totalCount etc similar to https://grafana.com/docs/grafana/latest/http_api/library_element/#get-all-library-elements ?